src/iap/shared/validators/is-login-unique.validator.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { AbstractControl, AsyncValidator, ValidationErrors } from '@angular/forms';
+import { catchError, map, Observable, of } from 'rxjs';
```

Comment:
:point_right: :point_left: 

```suggestion
import { AbstractControl, AsyncValidator, ValidationErrors } from '@angular/forms';

import { catchError, map, Observable, of } from 'rxjs';
```